Makefile

```diff
@@ -84,7 +84,7 @@ __check_defined = \
 .PHONY: $(GOPATH)/bin/minishift$(IS_EXE)
 $(GOPATH)/bin/minishift$(IS_EXE): $(ADDON_ASSET_FILE) vendor
 	go install -tags "$(BUILD_TAGS)" -pkgdir=$(ADDON_BINDATA_DIR) -ldflags="$(VERSION_VARIABLES)" ./cmd/minishift
-vendor:
+vendor: $(GOPATH)/bin/dep
 	dep ensure -v
```

Comment:
@LalatenduMohanty Could you add Add `echo $(shell dep version)` here to see which version of dep is installed?